<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QSkinEditorDialog</class>
 <widget class="QDialog" name="QSkinEditorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>681</width>
    <height>250</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Skin Editor</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="windowFrameTopLeft">
     <property name="cursor">
      <cursorShape>SizeFDiagCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/Resource/blau/frameTopLeft.png);
min-height: 14px;
max-height: 14px;
min-width: 14px;
max-width: 14px;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QFrame" name="windowFrameTopRight">
     <property name="minimumSize">
      <size>
       <width>10</width>
       <height>14</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>10</width>
       <height>14</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>SizeBDiagCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/Resource/blau/frameTopRight.png);
min-width: 10px;
max-width: 10px;
min-height: 14px;
max-height: 14px;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0" rowspan="3">
    <widget class="QFrame" name="windowFrameLeft">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>14</width>
       <height>0</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>SizeHorCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">min-width: 14px;
max-width: 14px;
border-image: url(:/Resource/blau/frameLeft.png);
border-top: 20px;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="2" rowspan="3">
    <widget class="QFrame" name="windowFrameRight">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>10</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>10</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>SizeHorCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/Resource/blau/frameRight.png);
border-top: 20px;
min-width: 10px;
max-width: 10px;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QFrame" name="windowFrameBottomLeft">
     <property name="minimumSize">
      <size>
       <width>14</width>
       <height>8</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>14</width>
       <height>8</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>SizeBDiagCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/Resource/frameBottomLeft.png);
border-image: url(:/Resource/blau/frameBottomLeft.png);
min-width: 14px;
max-width: 14px;
min-height: 8px;
max-height: 8px;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QFrame" name="windowFrameBottom">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>8</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>8</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>SizeVerCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/Resource/blau/frameBottom.png);
border-right: 30px;
min-height: 8px;
max-height: 8px;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QFrame" name="windowFrameBottomRight">
     <property name="minimumSize">
      <size>
       <width>10</width>
       <height>8</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>10</width>
       <height>8</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>SizeFDiagCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/Resource/blau/frameBottomRight.png);
min-width: 10px;
max-width: 10px;
min-height: 8px;
max-height: 8px;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QWidget" name="widgetContents" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="layoutContents">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="margin">
         <number>4</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Name</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEditSkinName"/>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineEditSkinAuthor"/>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="lineEditSkinVersion"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Author</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Version</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0" colspan="2">
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="pushButtonSaveSkin">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Save Skin</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="3" column="1">
         <widget class="QPlainTextEdit" name="plainTextEditSkinDescription">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>80</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Description</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1" rowspan="2">
    <widget class="QFrame" name="windowFrameTop">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>32</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>32</height>
      </size>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame#windowFrameTop {
	border-image: url(:/Resource/blau/frameTop.png);
	min-height: 32px;
	max-height: 32px;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="horizontalSpacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayoutTitlebar">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="1" rowspan="2">
         <widget class="QFrame" name="titlebarFrame">
          <property name="styleSheet">
           <string notr="true">QFrame#titlebarFrame {
	border-image: url(:/Resource/blau/frameStatusbar.png);
	border-top: 2px;
	border-bottom: 16px;
	border-right: 10px;
	padding-bottom: -4px;
}</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="spacing">
            <number>2</number>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <widget class="QToolButton" name="windowIcon">
             <property name="contextMenuPolicy">
              <enum>Qt::CustomContextMenu</enum>
             </property>
             <property name="styleSheet">
              <string notr="true">border: 0px solid transparent;</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>:/Resource/qtlogo-64.png</normaloff>:/Resource/qtlogo-64.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="windowText">
             <property name="contextMenuPolicy">
              <enum>Qt::CustomContextMenu</enum>
             </property>
             <property name="styleSheet">
              <string notr="true">font-weight: bold;
font-size: 12px;</string>
             </property>
             <property name="text">
              <string>Skin Editor</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="3" rowspan="2">
         <widget class="QFrame" name="titlebarButtonsFrame">
          <property name="styleSheet">
           <string notr="true">/*These move the buttons up so they aren't displayed in the center of the titlebar
   Remove these to center your buttons on the titlebar.*/
QFrame#titlebarButtonsFrame {
	background-color: transparent;
}</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayoutTitlebarButtons">
           <property name="margin">
            <number>0</number>
           </property>
           <property name="spacing">
            <number>0</number>
           </property>
           <item row="0" column="0">
            <widget class="QToolButton" name="minimizeButton">
             <property name="minimumSize">
              <size>
               <width>26</width>
               <height>26</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>26</width>
               <height>26</height>
              </size>
             </property>
             <property name="toolTip">
              <string>Minimize</string>
             </property>
             <property name="styleSheet">
              <string notr="true">QToolButton {
	border: 0px solid transparent;
	border-image: url(:/Resource/blau/minButton.png);
	min-width: 26px;
	max-width: 26px;
	min-height: 26px;
	max-height: 26px;
}

QToolButton:hover {
	border-image: url(:/Resource/blau/minButtonH.png);
}

QToolButton:pressed {
	border-image: url(:/Resource/blau/minButtonP.png);
}

QToolButton:disabled {
	border-image: url(:/Resource/blau/minButtonD.png);
}</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QToolButton" name="maximizeButton">
             <property name="minimumSize">
              <size>
               <width>34</width>
               <height>26</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>34</width>
               <height>26</height>
              </size>
             </property>
             <property name="toolTip">
              <string>Maximize</string>
             </property>
             <property name="styleSheet">
              <string notr="true">QToolButton {
	border: 0px solid transparent;
	border-image: url(:/Resource/blau/maxButton.png);
	margin-left: 4px;
	margin-right: 4px;
	min-width: 26px;
	max-width: 26px;
	min-height: 26px;
	max-height: 26px;
}

QToolButton:hover {
	border-image: url(:/Resource/blau/maxButtonH.png);
}

QToolButton:pressed {
	border-image: url(:/Resource/blau/maxButtonP.png);
}

QToolButton:disabled {
	border-image: url(:/Resource/blau/maxButtonD.png);
}

QToolButton:checked { /*The button will be checked when maximized is set*/
	border-image: url(:/Resource/blau/restoreButton.png);
}

QToolButton:checked:hover {
	border-image: url(:/Resource/blau/restoreButtonH.png);
}

QToolButton:checked:pressed {
	border-image: url(:/Resource/blau/restoreButtonP.png);
}

QToolButton:checked:disabled {
	border-image: url(:/Resource/blau/restoreButtonD.png);
}</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QToolButton" name="closeButton">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>26</width>
               <height>26</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>26</width>
               <height>26</height>
              </size>
             </property>
             <property name="toolTip">
              <string>Close</string>
             </property>
             <property name="styleSheet">
              <string notr="true">QToolButton {
	border: 0px solid transparent;
	border-image: url(:/Resource/blau/closeButton.png);
	min-width: 26px;
	max-width: 26px;
	min-height: 26px;
	max-height: 26px;
}

QToolButton:hover {
	border-image: url(:/Resource/blau/closeButtonH.png);
}

QToolButton:pressed {
	border-image: url(:/Resource/blau/closeButtonP.png);
}

QToolButton:disabled {
	border-image: url(:/Resource/blau/closeButtonD.png);
}</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="iconSize">
              <size>
               <width>16</width>
               <height>16</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="2" rowspan="2">
         <spacer name="horizontalSpacerTitlebar">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
